docs/source/configuration.rst

```diff
@@ -60,6 +63,12 @@ The ``server`` section provides directives on binding and high level tuning.
         connection: /tmp/pygeoapi-process-manager.db  # connection info to store jobs (e.g. filepath)
         output_dir: /tmp/  # temporary file area for storing job results (files)
 
+    api_rules:  # optional API design rules to which pygeoapi should adhere
+        api_version: 1.2.3  # omit to use pygeoapi's software version
+        strict_slashes: true  # trailing slashes will not be allowed and result in a 404
+        url_prefix: 'v{api_major}'  # adds a /v1 prefix to all URL paths
```

Comment:
In theory, can this be any prefix (i.e. not starting with `v`)?

Reply:
@tomkralidis Correct. This is just an example. The docs elaborate on this.

Reply:
See [docs](https://github.com/geopython/pygeoapi/pull/1152/files#diff-e662b6389537d2a309e93ca2065f311d3f00156764a4835671300f6c4e95ec1dR274-R287) for what you can fill in here.